tests/forms_tests/tests/test_forms.py

```diff
@@ -3856,7 +3923,7 @@ def clean(self):
             '<form>'
             '<p><label for="id_username">Username:</label>'
             '<input id="id_username" type="text" name="username" maxlength="10" '
-            'required></p>'
+            'aria-describedby="id_username_helptext" required></p>'
```

Comment:
An interesting case here. If you have help text and an `id` it will now give you an `aria-describedby` on your widget. However, if you don't actually include the help text in your custom form template .... (I'd argue that's an error in your code but....)

Reply:
Agreed. Can we add it to the `t`?